docs/api/config.md

```diff
@@ -1,99 +1,152 @@
 }
 ```
 
-### `distDir`
-
-Type: `string`
-
-The path—either absolute or relative—to the production-ready webpage/webapp directory that will be bundled by Tauri.
-
-<div className="alert alert--info" role="alert">
-  The target directory <em>must</em> contain an index.html file.
-</div>
-
-### `devPath`
-
-Type: `string`
-
-Can be a path - either absolute or relative - to a folder or a URL (like a live reload server).
-
-### `beforeDevCommand`
-
-Type: `string`<br/>
-Default: `""`
-
-A command to run before starting Tauri in dev mode.
-
-### `beforeBuildCommand`
-
-Type: `string`<br/>
-Default: `""`
+## `ctx`
 
-A command to run before starting Tauri in build mode.
+<Properties rows={[
+  { property: "debug", type: "boolean", description: `Tells the CLI to build on debug mode instead of release. Can also be used as <code>tauri build --debug</code>.` },
+  { property: "exitOnPanic", type: "boolean", description: `Whether or not the webview should be killed if some Rust code error happened. Can also be used as <code>tauri dev --exit-on-panic</code> or <code>tauri dev -e</code>.` }
+]} />
 
-## ctx
-
-```js
+```js title=Example
 "ctx" {
-  "prod": false,
-  "dev": false,
-  "target": "",
   "debug": false,
-  "targetName": "",
   "exitOnPanic": false,
 }
 ```
 
-### `prod`
-
-Type: `boolean`<br/>
-Default:
-
-### `dev`
-
-Type: `boolean`<br/>
-Default:
-
-### `target`
-
-Type: `string`<br/>
-Default:
-
-### `debug`
-
-Type: `boolean`<br/>
-Default:
-
-### `targetName`
+## `tauri`
 
-Type: `string`<br/>
-Default:
-
-### `exitOnPanic`
-
-Type: `boolean`<br/>
-Default:
+<Properties rows={[
+  {
+    property: "embeddedServer", type: "object",
+    child: <Properties rows={[{ property: "active", type: "boolean", description: `Set it to <code>false</code> if you plan to serve your application statically.` }]} />
+  },
+  {
+    property: "bundle", type: "object",
+    child: <Properties rows={[
+      { property: "active", type: "boolean", description: `` },
+      { property: "targets", type: "string", description: `The name of the built application. If this is not present, then it will use the name value from your Cargo.toml file.` },
+      { property: "identifier", type: "string", description: `A string that uniquely identifies your application, in reverse-DNS form (for example, "com.example.appname" or "io.github.username.project"). For OS X and iOS, this is used as the bundle's CFBundleIdentifier value; for Windows, this is hashed to create an application GUID.` },
+      { property: "icon", type: "string[]", description: `A list of (relative to src-tauri) icon paths to use for your application bundle.` },
+      { property: "resources", type: "string[]", description: `A list of files or directories which will be copied to the resources section of the bundle. Globs are supported.` },
+      { property: "externalBin", type: "string[]", description: `A list of—either absolute or relative—paths to binaries to embed with your application.` },
```

Comment:
should mention that each item like `my-binary` will look for `my-binary-${targetTriple}.exe` on Windows and `my-binary-${targetTriple}` on mac/linux.